frontend/revive-gallery-app/src/__tests__/ProductForm.test.js

```diff
@@ -17,11 +12,7 @@ test('renders the product form', () => {
 });
 
 test('test the file formats', () => {
-  render(
-    <MemoryRouter> // MemoryRouter mimics BrowserRouter or HashRouter
-        <ProductForm />
-    </MemoryRouter>
-  );
+  render(<ProductForm />);
```

Comment:
@bkarakala This does not work if the component uses useNavigate hook. Please revert it back to MemoryRouter